lib/Dialect/LWE/IR/LWEOps.td

```diff
@@ -189,20 +189,20 @@ def LWE_RMulPlainOp : LWE_CiphertextPlaintextOp<"rmul_plain", [Commutative]> {
 
 def LWE_RNegateOp : LWE_Op<"rnegate", [SameOperandsAndResultType, ElementwiseMappable]> {
   let summary = "Negate a RLWE ciphertexts";
-  let arguments = (ins NewLWECiphertextLike:$input);
-  let results = (outs NewLWECiphertextLike:$output);
+  let arguments = (ins LWECiphertextLike:$input);
+  let results = (outs LWECiphertextLike:$output);
   let assemblyFormat = "operands attr-dict `:` type($output)";
 }
 
 def LWE_MulScalarOp : LWE_Op<"mul_scalar", [ElementwiseMappable,
     AllTypesMatch<["ciphertext", "output"]>]> {
   let summary = "Multiply an LWE ciphertext by a scalar";
-  let arguments = (ins NewLWECiphertextLike:$ciphertext, AnyInteger:$scalar);
-  let results = (outs NewLWECiphertextLike:$output);
+  let arguments = (ins LWECiphertextLike:$ciphertext, AnyInteger:$scalar);
+  let results = (outs LWECiphertextLike:$output);
   let hasFolder = 1;
 }
 
-def LWE_RLWEEncodeOp : LWE_Op<"rlwe_encode", [NewHasEncoding<"output", "encoding", "NewLWEPlaintextType">]> {
+def LWE_RLWEEncodeOp : LWE_Op<"rlwe_encode", [NewHasEncoding<"output", "encoding", "LWEPlaintextType">]> {
```

Comment:
Did you push that change already? Because I still see a bunch of `NewHasEncoding`?